.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "editor.wordWrap": "wordWrapColumn",
+    "[markdown]": {
+
+        "editor.wordWrap": "wordWrapColumn",
+        "editor.wordWrapColumn": 80,
+        "editor.quickSuggestions": false
+    }
+}
```

Comment:
While a good idea, we don't usually include editor-specific files in our repository.

Reply:
Hi, thanks so much. How can I roll that back? Is it possible?

Reply:
Yep! If you're using Git with the command line, you can run the following from the repository root to remove the file from git *without* removing it from your filesystem:

```sh
git rm --cached .vscode/settings.json
```

Then just `git add -u && git commit` to make a new commit without the file included :slightly_smiling_face:

Reply:
I've gone ahead and done this in 1a54190.